examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -1647,23 +1647,23 @@ server cluster IasZone = 1280 {
   readonly attribute int8u zoneId = 17;
   readonly global attribute int16u clusterRevision = 65533;
 
-  request struct ZoneEnrollRequestRequest {
+  request struct ZoneEnrollResponseRequest {
+    IasEnrollResponseCode enrollResponseCode = 0;
+    INT8U zoneId = 1;
+  }
+
+  response struct ZoneEnrollRequest {
     IasZoneType zoneType = 0;
     INT16U manufacturerCode = 1;
   }
 
-  request struct ZoneStatusChangeNotificationRequest {
+  response struct ZoneStatusChangeNotification {
```

Comment:
We seem to have removed ZoneEndrollResponse, however that response is used in the command ZoneEnrollREquest.

Also the name of the command argument is 'ZoneEnrollRequestRequest' and above the struct is defined with the wrong name.